site/src/pages/StarterTemplatePage/StarterTemplatePage.tsx

```diff
@@ -3,14 +3,12 @@ import { Helmet } from "react-helmet-async";
 import { useQuery } from "react-query";
 import { useParams } from "react-router-dom";
 import { templateExamples } from "api/queries/templates";
-import { useDashboard } from "modules/dashboard/useDashboard";
 import { pageTitle } from "utils/page";
 import { StarterTemplatePageView } from "./StarterTemplatePageView";
 
 const StarterTemplatePage: FC = () => {
   const { exampleId } = useParams() as { exampleId: string };
-  const { organizationId } = useDashboard();
-  const templateExamplesQuery = useQuery(templateExamples(organizationId));
+  const templateExamplesQuery = useQuery(templateExamples());
```

Comment:
...and then here, we could just do...

```suggestion
  const templateExamplesQuery = useQuery(templateExamples("default"));
```

Reply:
yes, this could work but it doesn't matter if there is no plans to have different example templates for each organization

Reply:
but the `API` module should strive to stick close to the shape of the actual API. it expects an organization ID, so the function should take an organization ID, imo. It's easy enough to just say `"default"` in the two places we use it.